ofrak_core/ofrak/core/free_space.py

```diff
@@ -360,22 +422,47 @@ async def modify(self, resource: Resource, config: FreeSpaceAllocation) -> None:
 
         for fs in wholly_allocated_resources:
             fs.resource.remove_tag(FreeSpace)
+            fs.resource.remove_tag(RuntimeFreeSpace)
+
+            # Without committing the changes to our tags here, removing both the child and
+            # parent tags will cause a KeyError on update.
+            await fs.resource.save()
+            fs.resource.remove_tag(AnyFreeSpace)
```

Comment:
see #507 for why this hack is needed

Reply:
Can this be removed now that #513 is merged, or is it still necessary?

Reply:
yes, removed the workaround